Source: written by Calvin Walton <calvin.walton@kepstin.ca>
Upstream: http://bugs.python.org/issue25229
Reason: distutils checks for 'gcc' in compiler name, but Exherbo uses 'cc'

diff -ru Python-3.4.3.orig/Lib/distutils/unixccompiler.py Python-3.4.3/Lib/distutils/unixccompiler.py
--- Python-3.4.3.orig/Lib/distutils/unixccompiler.py	2015-09-24 17:47:51.326840548 -0400
+++ Python-3.4.3/Lib/distutils/unixccompiler.py	2015-09-24 17:48:44.777604734 -0400
@@ -232,7 +232,7 @@
         elif sys.platform[:7] == "irix646" or sys.platform[:6] == "osf1V5":
             return ["-rpath", dir]
         else:
-            if self._is_gcc(compiler):
+            if sys.platform[:5] == "linux" or self._is_gcc(compiler):
                 # gcc on non-GNU systems does not need -Wl, but can
                 # use it anyway.  Since distutils has always passed in
                 # -Wl whenever gcc was used in the past it is probably
